wait.go

```diff
@@ -35,3 +38,31 @@ func WaitUntil(ctx context.Context, condition func(context.Context) (bool, error
 		}
 	}
 }
+
+func Wait(ctx context.Context, condition func(context.Context) (bool, error)) error {
+	done, err := condition(ctx)
+	if err != nil {
+		return err
+	}
+	if done {
+		return nil
+	}
+
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+
+	for {
+		select {
+		case <-c:
+			done, err := condition(ctx)
```

Comment:
This will run the function again when it gets the interrupt signal. If you want the signal to abort the command then it's best to use WaitUntil and set up a separate goroutine that cancels the context when the signal is received (though I think the CLI package may already do this)

Reply:
Ah yes, it already cancels it when an abort command is sent.